### ISSUE TITLE
Bounding box values as Float

### ISSUE THREAD

**Issue by DecentMakeover** [yhenon/pytorch-retinanet#8]
Most of the datasets have integer values as their bounding box co-ordinates like 56 43 22 77 etc, but for the open dataset challenge the bounding box co-ordinates are in float values like 0.0012, 0.4120, 0.1250, 0.2224 etc, so when during training my classification loss decreases but my regression loss stays 0, Any suggestion on what i could modify to make this work.For now in the dataloader script from line 274-277 i have changed the int argument to float to make it to accept float values.Thanks in advance

**Comment by DecentMakeover** [comment]
Oh ,let me try this out. 

And i havent been using the oid_dataset.py file, I will go through that too.
Thanks
